scripts/configure-dh.sh

```diff
@@ -0,0 +1,363 @@
+#!/bin/bash
+
+# Constants
+BACKSTAGE_CR_NAME="ai-rh-developer-hub"
+DEFAULT_RHDH_DEPLOYMENT="backstage-ai-rh-developer-hub" # deployment created by rhdh operator by default
+PLUGIN_CONFIGMAP="dynamic-plugins" # configmap created by rhdh operator for plugins by default
+EXTRA_ENV_SECRET="ai-rh-developer-hub-env" # secret created by rhdh installer to store private env vars
+
+# Variables
+BASE_DIR="$(realpath $(dirname ${BASH_SOURCE[0]}))/.."
+CATALOGS_FILE=${CATALOGS_FILE:-"${BASE_DIR}/catalogs.yaml"}
+RHDH_DEPLOYMENT="${DEFAULT_RHDH_DEPLOYMENT}"
+RHDH_EXTRA_ENV_SECRET="${EXTRA_ENV_SECRET}"
+RHDH_PLUGINS_CONFIGMAP="${PLUGIN_CONFIGMAP}"
+NAMESPACE=${NAMESPACE:-"ai-rhdh"}
+EXISTING_NAMESPACE=${EXISTING_NAMESPACE:-''}
+EXISTING_DEPLOYMENT=${EXISTING_DEPLOYMENT:-''}
+EXISTING_EXTRA_ENV_SECRET=${EXISTING_EXTRA_ENV_SECRET:-''}
+RHDH_PLUGINS=${RHDH_PLUGINS:-''}
+RHDH_INSTANCE_PROVIDED=${RHDH_INSTANCE_PROVIDED:-false}
+RHDH_GITHUB_INTEGRATION=${RHDH_GITHUB_INTEGRATION:-false}
+RHDH_GITLAB_INTEGRATION=${RHDH_GITLAB_INTEGRATION:-false}
```

Comment:
I think both cant be defaulted to false, one should be true. And I think Github should default to true

Reply:
We also didnt set either of these in configure.sh; so all those steps were missed.

Reply:
@maysunfaisal Should be resolved with https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/10/commits/40446e43b9b7d40d8f894e8eb8ee7ba8231e1b50